docs/guide/ecosystem/server-side-rendering.md

```diff
@@ -1,22 +1,110 @@
 # Server Side Rendering
 
-## When and why to use SSR
+Server side rendering can be explained as moving the process of rendering pages of an SPA from the client's browser to the server. This process is similar to what we used to do a few years ago, where we had server side languages like PHP, Java, Python render the page and return an HTML response.
+
+The difference is, that once the page gets loaded by the client's browser, the JavaScript framework, Vue in our case, takes over the application's routing and rendering processes, giving the impression of a faster navigation and smoother experience.
+
+#### Useful Resources
+
+* [The Benefits of Server Side Rendering Over Client Side Rendering](https://medium.com/walmartlabs/the-benefits-of-server-side-rendering-over-client-side-rendering-5d07ff2cefe8) - a very well written explanation of the benefits of SSR, from the Wallmart developer team.
+* [What’s Server Side Rendering and do I need it?](https://medium.com/@baphemot/whats-server-side-rendering-and-do-i-need-it-cb42dc059b38)
+* [When to use Server-Side Rendering (SSR) in Vue.js projects](https://codeburst.io/when-to-use-server-side-rendering-ssr-in-vue-js-projects-697bd925d57b)
+* [What is React Server Side Rendering and should I use it?](https://dev.to/mladenstojanovic/what-is-react-server-side-rendering-and-should-i-use-it-5b7i) - For React, by the concepts are the same.
+
+## Benefits of SSR
+
+Server side rendering solves a few of the most annoying problems of Single-page applications:
+
+### SEO friendly 
+
+Most crawlers cannot scan websites that are rendered with JavaScript, resulting search engines registering empty pages without content. This directly leads to bad search engine ranking. 
```

Comment:
Right, most crawlers can't interpret JS. Google can, for example, but that doesn't mean that it'll rank the page high as it takes substantial time to load anyway. Might be worth to mention.

Reply:
Correct. It renders with an older Chrome version, which causes weird things for none transpiled code.